index.html

```diff
@@ -53,5 +53,144 @@ <h2 class=" role"> I am a Software developer</h2>
         
     </div>
     
+    <section class ="worksection">
```

Comment:
- [ ] Please make sure that the mrgin between the paragraph and the tech list matches the design 
your code 
![image](https://user-images.githubusercontent.com/74744787/212490832-dec6eb56-09b7-421d-a2e7-d55d77f81425.png)
Figma 
![image](https://user-images.githubusercontent.com/74744787/212490850-d0c14dfe-b75e-4800-9471-e02e8260b03b.png)

- [ ] Please fix the margin between the work section and the about me section
![image](https://user-images.githubusercontent.com/74744787/212490936-9ffccb4c-66ac-4ada-a98e-47f8bec66ae0.png)
the design 
![image](https://user-images.githubusercontent.com/74744787/212490952-194b0bbf-cbce-4b77-be84-e00bd3451d52.png)